grails-app/views/common/_menuicons.gsp

```diff
@@ -41,7 +41,7 @@
                                     <span class="subsection-title">${subsection?.label}</span>
                                 </g:if>
                                 <g:each in="${subsection?.menuItems}" var="item">
-                                    <a href="${item?.href}" class="dropdown-item">
+                                    <a href="${item?.href}" class="dropdown-item" role="menuitem">
```

Comment:
how is this `role="menuitem"` working?

Reply:
Yes, E2E tests rely on the role selectors as well. Half of our menu already had this role, I am adding it to the missing parts so the tests can be unified.